src/Core/Services/Implementations/LaunchDarklyFeatureService.cs

```diff
@@ -190,15 +195,24 @@ void SetCommonContextAttributes(ContextBuilder builder)
 
             case Identity.ClientType.Organization:
                 {
-                    if (_currentContext.OrganizationId.HasValue)
+                    ContextBuilder ldOrg;
```

Comment:
💭 Not sure if this override belongs here. If someone was in an organization context already this doesn't really make sense to use, but would code be passing the override in? I guess that would have no effect right?